cea/demand/demand_writers.py

```diff
@@ -25,121 +21,171 @@ class DemandWriter(object):
     - implement the `write_to_csv` method
     """
 
-    def __init__(self, gv):
-        self.gv = gv
-        self.vars_to_print = gv.demand_building_csv_columns
+    def __init__(self, loads, massflows, temperatures):
+        if loads == []:
+            self.LOAD_VARS = ['QEf', 'QHf', 'QCf', 'Ef', 'Qhsf', 'Qhs', 'Qhsf_lat', 'Qwwf', 'Qww', 'Qcsf',
```

Comment:
using uppercase implies that these are constants, but they're not. we could refactor/rename them to be private variables (e.g.  `self._load_vars`, note the leading underscore. or just `self.load_vars`)

Reply:
fixed